src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -3357,6 +3403,7 @@ class ParquetFileFormatTypeParameters(BaseSegment):
         Sequence(
             "SNAPPY_COMPRESSION", Ref("EqualsSegment"), Ref("BooleanLiteralGrammar")
         ),
+        Sequence("BINARY_AS_TEXT", Ref("EqualsSegment"), Ref("BooleanLiteralGrammar")),
```

Comment:
I wonder if all the ones like this:
```
        Sequence("...something...", Ref("EqualsSegment"), Ref("BooleanLiteralGrammar")),
```

could be combined into something like:
```
        Sequence(
            OneOf("SNAPPY_COMPRESSION", "BINARY_AS_TEXT", ...),
            Ref("EqualsSegment"),
            Ref("BooleanLiteralGrammar")),
```

No real concern, just seems tidier.

Reply:
Yes, it works. I can make these changes if you'd like.

Reply:
Sure, if you don't mind.

Reply:
Done! Also for the similar statements in all file format types